.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "yaml.schemaStore.enable": false,
```

Comment:
This disables the yaml extension's ability to look up schemas for yaml files, because the one it resolves for github actions files is just plain wrong, which makes it hard to author the file correctly. Instead I added a github actions-specific extension for intellisense and schema support.